homeassistant/components/climate/__init__.py

```diff
@@ -303,41 +303,6 @@ class ClimateEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_ATTR_):
 
     __climate_reported_legacy_aux = False
 
-    __mod_supported_features: ClimateEntityFeature = ClimateEntityFeature(0)
-    # Integrations should set `_enable_turn_on_off_backwards_compatibility` to False
-    # once migrated and set the feature flags TURN_ON/TURN_OFF as needed.
-    _enable_turn_on_off_backwards_compatibility: bool = True
```

Comment:
Does it make sense to cleanup all the integrations that have `_enable_turn_on_off_backwards_compatibility = False`?

Reply:
Just want to make sure it's not forgotten... and since there's only one...

Reply:
Actually there is 183 occurrences, that's why I would do it separately in chunks.